editor/plugins/editor_resource_tooltip_plugins.h

```diff
@@ -49,22 +49,22 @@ class EditorResourceTooltipPlugin : public RefCounted {
 	static void _bind_methods();
 
 	GDVIRTUAL1RC(bool, _handles, String)
-	GDVIRTUAL2RC(Object *, _make_tooltip_for_path, String, Dictionary)
+	GDVIRTUAL3RC(Object *, _make_tooltip_for_path, String, Dictionary, Object *)
```

Comment:
It should be possible to use `Control *` here. I can see it being used by other editor plugin code, for example.